src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Removing this line in your demo in dev tools doesn't change anything for me, I think it's redundant.